.travis.yml

```diff
@@ -1,9 +1,11 @@
 language: python
 python:
  - 2.7
+ - 3.6
```

Comment:
Or, we can remove the 2.7, and run them all in 3.6 -- I see travis fails with "no 3.6 found" when running under 2.7.